jylee0805/momoCall
# Feature/Message func update

[Website URL](https://momocall--staging-chat-vj5jwzs8.web.app/chat?member=ZxfYfZkr7swTpLMZSAiH&product=201807242211)

## Description
- Complete message sending to different chatrooms
- Add fixed response database

## Test Plan
1. Open the Website URL
2. Change the path to `/chat?member=shopId&product=productNumber`, `/chat?member=shopId&order=orderNumber` to ensure the page content is correct.
3. Input specific questions and observe if the correct answers are returned.
